practice/practice-02/src/styles/components/_card-item.scss

```diff
@@ -0,0 +1,84 @@
+.card-item {
+  @include flex-container(true, $align-items: center, $flex-direction: column);
+  @include size(auto, 516px);
+
+  position: relative;
+}
+
+.food-image {
+  @include setAbsoluteAndLocation(true, $top: 29px);
+
+  margin: auto;
+}
+
+.price {
+  @include setAbsoluteAndLocation(true, $top: 176px, $right: 70px);
+  @include size(52px, 52px);
+
+  color: $white;
+  border: 4px solid $white;
+  border-radius: 50px;
+  background-color: $secondary-color;
+  font-size: $fs-md;
+  text-align: center;
+  line-height: 50px;
+}
+
+.content-card {
+  @include setAbsoluteAndLocation(true, $bottom: 60px);
+  @include size(100%, auto);
+  @include flex-container(true, $align-items: center, $flex-direction: column);
+}
+
+.rate {
+  @include flex-container(true, $align-items: center, $gap: 10px);
+}
+
+.avatar-container {
+  position: relative;
+  @include size(74px, 38px);
+}
+
+.avatar {
+  &-0 {
```

Comment:
Should not use index like this, pls check first-child, last-child, n-child on css

Reply:
Fixed.